From 000a214b9c4c430e7e70e5e2677faa96de38548c Mon Sep 17 00:00:00 2001
Message-ID: <000a214b9c4c430e7e70e5e2677faa96de38548c.1772636661.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Wed, 4 Mar 2026 16:01:26 +0100
Subject: [PATCH] geometry.c: include system.h

HAVE_STDCKDINT_H determines whether fz_ckd_add_* is defined via macros
and builtins or implemented. Include system.h in geometry.c so that it
knows HAVE_STDCKDINT_H.

This fixes a FTBFS during mupdfwrap.py -b 01.
---
 source/fitz/geometry.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/source/fitz/geometry.c b/source/fitz/geometry.c
index 95c24a9c8..9a4ec3edd 100644
--- a/source/fitz/geometry.c
+++ b/source/fitz/geometry.c
@@ -21,6 +21,7 @@
 // CA 94129, USA, for further information.
 
 #include "mupdf/fitz.h"
+#include "mupdf/fitz/system.h"
 
 #include <math.h>
 #include <float.h>
-- 
2.53.0.766.gc91a775c12

